KDIC/PlayerViewController.swift

```diff
@@ -6,17 +6,58 @@ class PlayerViewController: UIViewController {
   @IBOutlet weak var titleLabel: UILabel!
   @IBOutlet weak var subtitleLabel: UILabel!
   @IBOutlet weak var imageView: UIImageView!
-  
+
   @IBAction func togglePlayerButtonWasTapped(_ sender: Any) {
     KDICPlayer.toggle()
   }
+    
   override func viewDidLoad() {
     super.viewDidLoad()
+    
+    self.imageView.image = UIImage(named: "Equalizer")
+    
+    button.setImage(imgShell1, forState:.Normal);
```

Comment:
Does this work? Where is `imgShell1` coming from? Same for `coin` in the next line.